os/kernel/log_dump/log_dump.c

```diff
@@ -47,11 +51,14 @@
 #endif
 
 #define LOG_DUMP_OK			 0
+#define LOG_DUMP_OK_FSZ			 1
 #define LOG_DUMP_MEM_FAIL		-1
 #define LOG_DUMP_HEAP_FAIL		-2
 #define LOG_DUMP_SIZE_FAIL		-3
+#define LOG_DUMP_STR_FREEMEM		15
```

Comment:
Could you rebase the latest master?
I add start and stop functionality before(https://github.com/Samsung/TizenRT/commit/c8399b6af9bce9114947977bef798e6d6887b183), so there was a change in this file also.